test/functional/fixtures/runner/browser-dialogs/basepage.test.js

```diff
@@ -0,0 +1,31 @@
+'@fixture basePage';
```

Comment:
I'd like to rename the file to `before-unload-diaog.test.js` if we will create separate files for other dialogs or `index.test.js` (if we will have just one test file here)